internal/client/config.go

```diff
@@ -117,26 +117,27 @@ func locateConfigFile() string {
 	return filePath
 }
 
+// All the errors returned by the helper methods called in this function get ignored (down the road we throw an error when all auth methods have failed.) We only use these errors to log warnings to the user.
 func readCliConfigFile(configFilePath string) CLIHostConfig {
 	config := CLIHostConfig{}
 
 	// Read the CLI config file content.
 	content, err := os.ReadFile(configFilePath)
 	if err != nil {
-		log.Printf("[ERROR] Error reading CLI config or credentials file %s: %v", configFilePath, err)
+		log.Printf("[WARN] Error reading CLI config or credentials file %s: %v", configFilePath, err)
```

Comment:
@uturunku1 Sweet. I'd recommend updating the language to not use an alarming word like `Error` unless we are logging an `[Error]`.
```
log.Printf("[WARN] Unable to read CLI config or credentials file %s: %v", configFilePath, err)
```
Same with other lines.

Reply:
sounds good! I will make this change